packages/connection-ui/webpack.config.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+const getBaseWebpackConfig = require( '@automattic/calypso-build/webpack.config.js' );
+const DependencyExtractionWebpackPlugin = require( '@wordpress/dependency-extraction-webpack-plugin' );
+const path = require( 'path' );
+
+const isDevelopment = process.env.NODE_ENV !== 'production';
+
+const baseConfig = getBaseWebpackConfig(
+	{ WP: false },
```

Comment:
The `env.WP` flag does two things:
1. Initializes the `DependencyExtractionWebpackPlugin` to not include the dependencies available in WordPress into the build. We do that separately by including the plugin manually in that config.
2. Creates an abstraction layer using `@wordpress/element`. I don't see any potential need for it, so I think it would be an excessive complexity. We can still use the `@wordpress/element` components (e.g. `Button`) when needed.

This is the way we handle things in Jetpack, this logic is copied from there.